.version

```diff
@@ -1 +1 @@
-v2.0
+v2.1
```

Comment:
Need to update README to mention this version (v2.1).

Reply:
Thank you.